services/case-ms/src/lambdas/updatePdf.ts

```diff
@@ -47,7 +59,7 @@ export async function main(event, context) {
   return true;
 }
 
-function updateCasePdfAttributes(caseKeys, pdf) {
+function updateCasePdfAttributes(caseKeys: CaseKeys, pdf: string) {
```

Comment:
Option:
If the args have a clear type hinting, could the args itself be trimmed to just keys? :)

Reply:
Done